src/components/NodePackageDownloads/index.css

```diff
@@ -203,4 +230,20 @@
 .show-versions-button:hover {
   background-color: var(--ifm-link-color);
   color: var(--ifm-footer-background-color);
-}
\ No newline at end of file
+}
+
```

Comment:
@chanderprakash20 , can you please remove this unnecessary spaces below, also format the page if any

Reply:
I noticed that it is applied in other PR, thanks!